wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -34,9 +34,9 @@ When you launch OBS Studio for the first time, the auto-configuration wizard wil
 
 ![](img/configuration-wizard.png "OBS Studio's auto-configuration wizard")
 
-The next section to be configured are video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
+The next section to configure contains video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
```

Comment:
```suggestion
The next step is to configure some video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
```

Reply:
Subjective, I tried to keep it closer to the original, but your suggestion is better.

Reply:
it doesn't have to be "close to the original" - if you rewrote the entire article but better that'd be fine too